heft-plugins/heft-sass-plugin/heft-plugin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/heft-plugin.schema.json",
+
+  "taskPlugins": [
+    {
+      "pluginName": "SassTypingsPlugin",
```

Comment:
This plugin only handles typings? It doesn't handle SCSS->CSS yet?

Reply:
That seems to be the case, yes. Let's sync offline to see how we can handle/deal with SASS.

Reply:
It does the SCSS -> CSS conversion as well. The name of all the methods implied it didn't though. I'll update but leave the existing names, since I need to pull from main into the parent branch anyway (where a lot of the renames have already occurred)